cmd/kola/options.go

```diff
@@ -126,6 +129,17 @@ func syncOptions() error {
 		return fmt.Errorf("unsupport platform %q", kolaPlatform)
 	}
 
+	ok = false
```

Comment:
I'd break this out into a function like `validateDistro` just to keep this function as clean as possible and hide the ugly golang away